deftree/build_deftree.go

```diff
@@ -92,30 +92,30 @@ func New(req *plugin.CodeGeneratorRequest, serviceFile io.Reader) (Deftree, erro
 	return &dt, nil
 }
 
-func NewFromString(def string) (Deftree, error) {
+func NewFromString(def, GOPATH string) (Deftree, error) {
```

Comment:
In the go standard library, they treat the "$GOPATH" this way.
```
/usr/local/go/src/go/build/build.go
...
	GOOS        string // target operating system
	GOROOT      string // Go root
	GOPATH      string // Go path
	CgoEnabled bool // whether cgo can be used
```

https://github.com/golang/go/blob/master/src/go/build/build.go#L34

If it really is a big deal. I could go `goPath`, but go does treat the "$GOPATH" standard library like this and I like going with it.